static/js/stream_ui_updates.js

```diff
@@ -70,8 +70,19 @@ export function update_toggler_for_sub(sub) {
         }
         stream_edit.toggler.disable_tab("personal_settings");
     }
+    enable_or_disable_subscribers_tab(sub);
 }
 
+export function enable_or_disable_subscribers_tab(sub) {
+    if (sub.invite_only) {
+        if (sub.subscribed) {
+            stream_edit.toggler.enable_tab("subscriber_settings");
+        }
+        if (!sub.subscribed) {
+            stream_edit.toggler.disable_tab("subscriber_settings");
+        }
+    }
+}
```

Comment:
We currently allow admins to access subscribers for all streams including unsubscribed private streams.